package.json

```diff
@@ -19,11 +19,12 @@
     "async": "0.9.0",
     "babyparse": "0.2.1",
     "commander": "2.5.0",
+    "common": "0.2.5",
```

Comment:
@ianjorgensen added common in order to test but we should really just be using the already imported async